src/Microsoft.Win32.SystemEvents/src/Microsoft.Win32.SystemEvents.csproj

```diff
@@ -155,7 +155,6 @@
     <Reference Include="System.Collections" />
     <Reference Include="System.ComponentModel.EventBasedAsync" />
     <Reference Include="System.ComponentModel.Primitives" />
-    <Reference Include="System.ComponentModel.TypeConverter" />
```

Comment:
It seems like this is what is causing the All Configurations to fail. The reason is because we build `Microsoft.Win32.SystemEvents` for `netcoreapp2.0` and `netcoreapp`. In `netcoreapp` we no longer need to depend on `System.ComponentModel.TypeConverter` because `InvalidAsynchronousStateException` was moved to `System.ComponentModel.Primitives` however in `netcoreapp2.0` that shipped a while ago, this type still lives in `System.ComponentModel.TypeConverter` so, instead of deleting this reference, just condition it to be included when `TargetGroup` == `netcoreapp2.0`

i.e:

```xml
<Reference Include="System.ComponentModel.TypeConverter" TargetGroup="'$(TargetGroup)' == 'netcoreapp2.0'" />
```